### ISSUE TITLE
Contrasts of colours makes it hard to read email address

### ISSUE THREAD

**Issue by bernardbaker** [forem/forem#7600]
**Describe the bug**

I was viewing somebody who placed the heart on my article. When I went to their profile I couldn't read the email address. Because of the contrast of colours.

**To Reproduce**

1. Go to profile in the picture.

![Screenshot_20200429-205512](https://user-images.githubusercontent.com/13556172/80641144-5374a880-8a5c-11ea-85bd-76a05c39c049.png)

2. Read the email address.

**Expected behavior**

I should be able to read the email address.

**Screenshots**

**Desktop (please complete the following information):**

- OS:
- Browser:
- Version:

**Smartphone (please complete the following information):**

- Device: pixel 3 xl
- OS: android
- Browser: dev.to app
- Version: n/a

**Additional context**

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Il looks like this depends on which colors the user has set:

https://github.com/thepracticaldev/dev.to/blob/c734457a4447520c5cc30a9d41496b4ef22ce5c7/app/views/articles/_user_metadata.html.erb#L2-L4

but as we know from the discussion here https://github.com/thepracticaldev/dev.to/issues/7434 that logic fails to take into account theming.

**Comment by rhymes** [comment]
I believe this was fixed by https://github.com/forem/forem/pull/8406

@bernardbaker let us know otherwise
